.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: python
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-script: py.test
\ No newline at end of file
+env:
+  - TOXENV=py27
+  - TOXENV=py34
+install:
+  - pip install tox
+script: tox
+matrix:
+  include:
+    - python: "3.5"
+      env: TOXENV=py35
```

Comment:
Moving the `env` section into the matrix would probably let you handle the language differences, doing e.g.

```
matrix:
  include:
    - python: "2.7"
      env: TOXENV=py27
    - python: "3.4"
      env: TOXENV=py34
    - python: "3.5"
      env: TOXENV=py35
```
although I'm not certain what performance impact this would have, or if it's actually a good idea.